torchx/lib/torchx/backend.ex

```diff
@@ -979,6 +979,43 @@ defmodule Torchx.Backend do
     |> to_nx(out)
   end
 
+  @impl true
+  def window_max(out, tensor, window_dims_tuple, opts) do
+    if Enum.any?(opts[:padding], fn conf -> conf |> Tuple.to_list() |> Enum.any?(&(&1 != 0)) end) do
+      raise ArgumentError, "padding unsupported"
+    end
+
+    strides = opts[:strides]
+
+    if tuple_size(window_dims_tuple) > 3 do
+      raise ArgumentError, "cannot have more than 3 dimensions for the window"
+    end
+
+    window_dilations = opts[:window_dilations]
+
+    strides = List.duplicate(1, 3 - length(strides)) ++ strides
+    window_dilations = List.duplicate(1, 3 - length(window_dilations)) ++ window_dilations
+
+    window_dims =
+      List.duplicate(1, 3 - tuple_size(window_dims_tuple)) ++ Tuple.to_list(window_dims_tuple)
+
+    pool_shape =
+      List.to_tuple(
+        List.duplicate(1, 4 - tuple_size(tensor.shape)) ++ Tuple.to_list(tensor.shape)
+      )
+
+    padding = List.duplicate(0, 3)
```

Comment:
why not create a static variable?

Reply:
This should be removed soon, and it's not like it's a costly operation to justify inlining it